code/_onclick/hud/human.dm

```diff
@@ -236,6 +236,8 @@
 		mymob.internals = new /obj/screen()
 		mymob.internals.icon = ui_style
 		mymob.internals.icon_state = "internal0"
+		if(istype(target.internal, /obj/item/tank)) //Internals on already? Iight, prove it
+			mymob.internals.icon_state = "internal1"
```

Comment:
```suggestion
		mymob.internals.icon_state = "internal[target?.internal? 1 : 0]"
```

Reply:
test it though, i don't know for sure if this'll work

Reply:
Tested and works fine. Committing now.